docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -231,29 +231,34 @@ V&V Checks:
-  * - 4
-    - XX relative risk on incidence of hemorrhage and obstructed labor 
+  * - 4a
+    - Relative risk of c-section on incidence of hemorrhage
+    - 2.05 (1.84-2.29)
+    - `Pubu et al 2021 <https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7887994/>`_ 
+    - This value is a stand-in from this population-based study in Tibet (see 'Source' column), in which the authors reported an odds ratio rather than a relative risk. With further research and analysis we will likely update this value. Outstanding items: how does c-section need overlap with hemorrhage/OL, what is the RR, how will we implement this with overlaps in total MD impact of facility type
+  * - 4b
+    - Relative risk of c-section on incidence of obstructed labor 
     - 
     - 
-    - Outstanding items: how does c-section need overlap with hemorrhage/OL, what is the RR, how will we implement this with overlaps in total MD impact of facility type 
+    - Outstanding items: how does c-section need overlap with hemorrhage/OL, what is the RR, how will we implement this with overlaps in total MD impact of facility type  
   * - 5
-    - XX% of pregnancy receive in each delivery facility type
-    - 
-    - 
-    - Confirm understanding that all pregnancies can/should receive this
+    - % of pregnancy receive azithromycin in each delivery facility type
+    - At home (10%), in public/governmental hospital (41%), in private for-profit health facility (10%), private not-for-profit health facility (10%), country-specific governmental health facility (10%)  
+    - SARA (Ethiopia; Table 3.8.2)
+    - These are placeholder values and will be updated with further analysis. We want these to be location specific, please code accordingly.  
   * - 6
-    - XX relative risk of incidence of sepsis and other infections
-    - 
-    - 
-    - Outstanding items: what is the RR, how will we implement this with overlaps in total MD impact of facility type 
+    - Relative risk of azithromycin on incidence of sepsis and other infections
+    - 0.65 (0.55, 0.77)
+    - `Tita et al 2023 <https://www.ajog.org/article/S0002-9378(22)02210-4/fulltext#undfig1>`_ 
+    - Outstanding items: how will we implement this with overlaps in total MD impact of facility type 
   * - 7
-    - XX% of pre-term or known LBW pregnancies will receive, split by delivery facility type
-    - 
-    - 
-    - Outstanding items: data by delivery facility, is this for preterm, LBW, or both/combination; believe this only affected neonatal outcomes, confirm with BMGF
+    - % of pre-term or known LBW pregnancies will receive antenatal corticosteroids, split by delivery facility type
+    - At home (10%), in public/governmental hospital (2%), in private for-profit health facility (22%), and private not-for-profit health facility (4%), country-specific governmental health facility (10%)  
+    - EmONC (Ethiopia; Table 10.5.4A)
```

Comment:
What's the denominator in these percentages? E.g., preterm births? All births? Preterm or LBW births?

Reply:
The denominator is preterm or LBW births, I can add a note of that

Reply:
The exact wording from the source is "Percent distribution of cases of preterm and low birth weight babies according to 
client status at birth/admission, treatment, and outcome, by facility type and managing authority"

Reply:
I added a note about this and that we want these values to be location-specific, even though currently these are values for Ethiopia only.

Reply:
Ok great, thanks! And the numerator is the number of those that got antenatal corticosteroids?

Why is the public hospital percentage so low (2%)? I would have expected the percentage in any hospital (public or private) to be higher than the percentage for home birth (10%). And 10% for home births seems rather high...

Reply:
Yes, you're right. I think this is another reason to simplify the stratification of facilities to the 3 options of hospital (high-level), clinic (low-level), and home. In EmONC, hospitals show 12% of preterm/LBW pregnancies receiving ACS, while public/governmental health facilities more generally only showed 2%. I will update these values accordingly, but I can do so in a separate PR.

Reply:
Sounds good, thanks